### PR TITLE
Add ability to execute commands in tee mode

### PR DESCRIPTION
**Pull request by ssbarnea**
This is a required feature for use within molecule as the executed
commands may run for a long period of time and users want to see
live progress.

Fixes: #40

### CODE REVIEW

File: src/ansible_compat/runtime.py [hunk 1]
```diff
@@ -116,14 +117,19 @@ def clean(self) -> None:
...
     def exec(
-        self, args: Union[str, List[str]], retry: bool = False
+        self, args: Union[str, List[str]], retry: bool = False, tee: bool = False
     ) -> CompletedProcess:
         """Execute a command inside an Ansible environment.
 
         :param retry: Retry network operations on failures.
         """
+        if tee:
+            runner: Callable[..., CompletedProcess] = subprocess_tee.run
```
Comment (webknjaz) [inline src/ansible_compat/runtime.py:127]: Don't call it runner. By invoking it, it doesn't *just* create an object but also invokes commands. It must be a verb.

Comment (ssbarnea) [reply src/ansible_compat/runtime.py:127]: `run_func` ?

File: src/ansible_compat/runtime.py [hunk 2]
```diff
@@ -116,14 +117,19 @@ def clean(self) -> None:
             shutil.rmtree(self.cache_dir, ignore_errors=True)
 
     def exec(
-        self, args: Union[str, List[str]], retry: bool = False
+        self, args: Union[str, List[str]], retry: bool = False, tee: bool = False
```
Comment (webknjaz) [inline src/ansible_compat/runtime.py:120]: I'm not sure how I feel about seeing a million args added to this method. It seems to be exhaustive.

Comment (ssbarnea) [reply src/ansible_compat/runtime.py:120]: I doubt we will too many args here. In fact we already have less than original run, as the universal_newlines are implicit.

**Pull request merged**